packages/fluentui/e2e/tests/menuDismissOnItemClick-test.ts

```diff
@@ -0,0 +1,51 @@
+import { menuItemClassName } from '@fluentui/react-northstar';
+
+const menuItem = `.${menuItemClassName}`;
+
+const firstSubmenuItem = '#first';
+const secondSubmenuItem = '#second';
+
+describe('Dismiss Menu on Item Click', () => {
+  beforeEach(async () => {
+    await e2e.gotoTestCase(__filename, menuItem);
+  });
+
+  describe('Mouse interactions', () => {
+    it('Should close on click', async () => {
+      await e2e.expectCount(menuItem, 1);
+      await e2e.clickOn(menuItem);
+      await e2e.expectCount(menuItem, 3);
+      await e2e.clickOn(firstSubmenuItem);
+      await e2e.expectCount(menuItem, 1);
```

Comment:
```suggestion
      await e2e.expectCount(submenu, 0);
      
      await e2e.clickOn(menuItem);
      await e2e.expectCount(submenu, 1);
      
      await e2e.clickOn(firstSubmenuItem);
      await e2e.expectCount(submenu, 0);
```

Can we use a selector for submenu instead?